core/build.gradle

```diff
@@ -22,6 +22,6 @@ dependencies {
 }
 
 apply from: '../tasks.gradle'
-apply from: 'bintray.gradle'
+apply from: "publish.gradle"
```

Comment:
Should probably be consistent with the double vs single quotation marks 😄

Reply:
#triggered :D